From 5e52d47182f7a689e45528df4a155a33e7ca3f99 Mon Sep 17 00:00:00 2001
From: tstegema <tstegema@90b65887-3827-0410-9a23-83215b262276>
Date: Tue, 1 Mar 2022 16:59:35 +0000
Subject: [PATCH 0984/1000] MANTIS 0062095 Logging: missing MOR-Barcode in
 session data platform.idl add PlatformIF.dataReplay to forward MOR data to
 platform

git-svn-id: svn://localhost/SelfServiceCommon/trunk@1713 90b65887-3827-0410-9a23-83215b262276
---
 SelfServiceCommon/Massai/idl/Massai/platform.idl | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/SelfServiceCommon/Massai/idl/Massai/platform.idl b/SelfServiceCommon/Massai/idl/Massai/platform.idl
index dfe9585f..3142dca6 100644
--- a/SelfServiceCommon/Massai/idl/Massai/platform.idl
+++ b/SelfServiceCommon/Massai/idl/Massai/platform.idl
@@ -27,7 +27,7 @@
 
 module PlatformManagement
 {
-    const string interface_version = "5.8";
+    const string interface_version = "5.9";
 
     /** Platform interface for the device components. Via this interface the device
         components communicate with the platform manager internally. */
@@ -482,6 +482,7 @@ module PlatformManagement
             @param componentType - type of the component that received the data; e.g.: BCR for barcode reader
             @param data - data as defined in cuss types.idl(msgDataType, aeaDataType, svgDataType, ...) */
         DataHandling dataReceived(in string componentType, in any data);
+        DataHandling dataReplay(in string componentType, in any data);
 
         void setupVCompListener(in string appRef, in string compRef, in any action, in any eventAcquireFilter, in any eventListener, in any elud);
 
-- 
2.41.0.windows.1

